.circleci/config.yml

```diff
@@ -764,6 +664,3 @@ workflows:
       - e2e-pilot-auth-v1alpha3-v2: # auth+v1alpha3+v2
           requires:
             - build
-      - e2e-pilot-cloudfoundry-v1alpha3-v2:
-          requires:
-            - build
```

Comment:
did you mean to remove the auth one? CF is testing a different environment

Reply:
Merged it into one job - so we capture the test out and save 30 seconds or so.